src/main/kotlin/demo/services/OrganizationService.kt

```diff
@@ -0,0 +1,27 @@
+package demo.services
+
+import demo.models.people.Organization
+import demo.repositories.OrganizationRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class OrganizationService(@Autowired private val organizationRepository: OrganizationRepository) {
+    fun getOrganizations(): MutableIterable<Organization> = organizationRepository.findAll()
+
+    fun getOrganization(id: String): Optional<Organization> = organizationRepository.findById(id)
```

Comment:
Id organizacji to String?

Reply:
tak, sprawdz se w modelu panie rewjułer XD